flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
why restricting to a single arch?

Reply:
Only had the x86_64 binary in the flatpak yml at the time, should be fixed now (added x86 and aarch64 too).